src/Uno.UI/UI/Xaml/DragDrop/DragDropExtension.wasm.cs

```diff
@@ -30,9 +30,7 @@
 using Uno.UI.Xaml;
 using System.Diagnostics.CodeAnalysis;
 
-#if NET7_0_OR_GREATER
```

Comment:
We'll probably need to make another pass at this once @ebariche is done with CSP.

Reply:
Sure. @ebariche Let me know once the work is done and I'll do it.